ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -440,7 +505,17 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
         guard let newWindow else {
             return
         }
+
         tokenSet.update(newWindow.fluentTheme)
+        titleView.tokenSet.update(newWindow.fluentTheme)
+        titleView.tokenSet.setOverrides(from: tokenSet, mapping: [
```

Comment:
Shouldn't these overrides be present inside `updateColors`?

Reply:
These override the tokens themselves (including both font and color tokens). `updateColors` just grabs the token values and passes them along.

Reply:
Then shouldn't these overrides be in the `registerOnUpdate` callback?

Reply:
I don't believe we have any precedent for where this should go. This probably warrants further discussion, but I still think this could be fine as is.

Reply:
There's precedent in `SideTabBar`, `TabBarView` and `BadgeView` which add their sub-controls' overrides in the callback. I went ahead and made the change.